build.gradle

```diff
@@ -6,6 +6,10 @@ buildscript {
         junitVersion = '4.12'
```

Comment:
should we consider removing JUnit4 in the near future?

Reply:
probably a good idea.  Junit5 isn't different enough to cause people problems.